sites/public/src/components/listing/ListingMap.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { GoogleMap, Marker, useJsApiLoader } from "@react-google-maps/api"
+import { Listing } from "@bloom-housing/backend-core/types"
+
+type ListingMapProps = {
+  listing: Listing
+  googleMapsHref: string
+}
+
+const containerStyle = {
```

Comment:
Plz update with strong typing that #68 adds as well plz (to avoid a build error that @chriscasto is getting but i am not )

```
const containerStyle: React.CSSProperties = {
```

Reply:
Ah that makes sense, I also had this issue where the position property gave a build error so I had removed it. After adding this, I'm able to add the position properly.